examples/decoding/plot_ssd_spatial_filters.py

```diff
@@ -153,19 +407,19 @@ def spectral_ratio_ssd(psd, freqs, freqs_sig, freqs_noise):
                                   fir_design='firwin'),
           filt_params_noise=dict(l_freq=freqs_noise[0], h_freq=freqs_noise[1],
                                   l_trans_bandwidth=1, h_trans_bandwidth=1,
-                                  fir_design='firwin'))
-
+                                  fir_design='firwin'), 
+          sampling_freq=raw.info['sfreq'], picks=picks)
+#%%
```

Comment:
these comments/directives should not be there. also see the ones below.

Reply:
Ok! I have a new version of this implementation, considering your comments and suggestions,  but keeping the cov. computation as in the CSP code, until you tell me what it best. How I updated the code?

Reply:
You are probably right because it is the way the thing is defined also in SPoC, now that I think of it. 
I suspect however, that results would look very similar while some time could be saved. We can try/benchmark at a later point in time. Let's first move on with the other points. Once that is done and we are happy we can add the SSD class to a proper module and write unit tests.